### PR TITLE
fix issus #1490

### PR DESCRIPTION
**Pull request by fanfuqiang**

### CODE REVIEW

File: arch/RISCV/RISCVInstPrinter.c [hunk 1]
```diff
@@ -93,7 +93,14 @@ void RISCV_printInst(MCInst *MI, SStream *O, void *info)
...
     	//NewMI = const_cast<MCInst *>(&UncompressedMI);
-  	if (/*NoAliases ||*/ !printAliasInstr(MI, O, info))
+	/* When Capstone enable print details, The riscv port will display no-consistency display
+	   for some alias instructions. for example `jalr x0, x1, 0` ALIAS `ret` in RISCV port.
+	   But `jalr x0, x1, 0` will only display `ret` and the details part only display
+	   `jalr`, This is because the LLVM do not need the details part like capstone, so LLVM 
+	   missing the operands process(x0, x1, 0), for LLVM print `ret` is enough.
+	   for now, when enable detail, we disable alias instructure.  */
+	// TODO: what should we print for `jalr x0, x1, 0` typs?
+    	if (MI->csh->detail || !printAliasInstr(MI, O, info))
```
Comment (aquynh) [inline arch/RISCV/RISCVInstPrinter.c]: hmm this does not make much sense to me: why this should depend on detail mode?

in other arch, we just try to print alias first, and fall thru to `printInstruction()` if that does not work.

**Comment by aquynh** [comment]
cool, but instead of adding this case to our sample code, we can instead put that as a test case in https://github.com/aquynh/capstone/blob/next/suite/cstest/issues.cs ?

File: arch/RISCV/RISCVInstPrinter.c (hunk 1 above)
Comment (fanfuqiang) [reply arch/RISCV/RISCVInstPrinter.c]: so you prefer output CASE A/B?
https://github.com/aquynh/capstone/issues/1490#issuecomment-496724793

**Comment by Rot127** [comment]
Thank you for the PR! I closed it because it is out of date. With the new [auto-sync update for v6](https://github.com/capstone-engine/capstone/blob/next/docs/cs_v6_release_guide.md) we made many changes to some main architectures and will do also to others.
This also changed the requirements we have now for new PRs.
 
If you still want to merge the changes, please rebase your fix onto the newest `next` branch and open a new PR.

**Pull request closed without merging**